osu.Game/Configuration/OsuConfigManager.cs

```diff
@@ -74,6 +74,7 @@ protected override void InitialiseDefaults()
 
             // Gameplay
             Set(OsuSetting.DimLevel, 0.3, 0, 1, 0.01);
+            Set(OsuSetting.DimLightness, 0, 0, 1, 0.01);
```

Comment:
"lightness" and "level" are synonymous. this should be called `DimColour` or `DimBaseColour` or something distinguishable. The same goes for the user-exposed description below.